terraform/gcp/projects/leap.tfvars

```diff
@@ -76,22 +76,26 @@ notebook_nodes = {
     machine_type : "n2-highmem-4",
   },
   # FIXME: Rename this to "n2-highmem-16" when given the chance and no such nodes are running
+  # FIXME: Remove node pool specific node_version pin when given the chance and no such nodes are running
   "medium" : {
     # A minimum of one is configured for LEAP to ensure quick startups at all
     # time. Cost is not a greater concern than optimizing startup times.
     min : 1,
     max : 100,
     machine_type : "n2-highmem-16",
+    node_version : "1.25.6-gke.1000",
   },
   "n2-highmem-64" : {
     min : 0,
     max : 100,
     machine_type : "n2-highmem-64"
   }
+  # FIXME: Remove node pool specific node_version pin when given the chance and no such nodes are running
```

Comment:
Removing the pin + terraform apply will delete the current node pool and recreate one with the newest k8s version? Or are extra steps needed to upgrade these machine types' k8s version?

Reply:
No removing it will actually not need to recreate the node pool, a rolling upgrade will be done to use the non-node-pool-specific version specified in terraform config above, with for 0 active nodes means its quick and a simple "update". But, only if no active users are on it.